src/platform/events/blockchain.ts

```diff
@@ -54,59 +55,73 @@ export namespace BidAcceptedEvent {
   export const validate: ValidateFunction<BidAcceptedEvent> = generateLazyValidator(schema)
 }
 
+type ItemEventMetadata = {
+  address: string
+  image: string
+  buyer: string
+  seller: string
+  category: string
+  rarity?: string
+  link: string
+  nftName?: string
+  tokenId: string
+  network: string
+  title: string
+  description: string
+}
+
 export type ItemSoldEvent = BaseEvent & {
   type: Events.Type.BLOCKCHAIN
   subType: Events.SubType.Blockchain.ITEM_SOLD
-  metadata: {
-    address: string
-    image: string
-    buyer: string
-    seller: string
-    category: string
-    rarity?: string
-    link: string
-    nftName?: string
-    tokenId: string
-    network: string
-    title: string
-    description: string
-  }
+  metadata: ItemEventMetadata
+}
+
+// TODO: should we use the itemId instead of tokenId?
```

Comment:
do you mean renaming the property?

Reply:
I think some teams refers to that id as `itemId` and some others as `tokenId`, yet, our ADR says Token Id, so I think we should build the language on top of that: https://adr.decentraland.org/adr/ADR-244